mabwiser/mab.py

```diff
@@ -701,7 +727,10 @@ def learning_policy(self):
             lp = self._imp
 
         if isinstance(lp, _EpsilonGreedy):
-            return LearningPolicy.EpsilonGreedy(lp.epsilon)
+            if issubclass(type(lp), _Popularity):
```

Comment:
Nice catch, though you could simplify the logic by checking for Popularity as the first option instead of sticking to the alphabetical order (if Popularity, elif EpsilonGreedy)

Reply:
I like that --let me assert that it works as you intended, and I will revise

Reply:
I checked that it works as you suggested with Popularity as the first option: 

        if isinstance(lp, _Popularity):
            return LearningPolicy.Popularity()
        elif isinstance(lp, _EpsilonGreedy):
            return LearningPolicy.EpsilonGreedy(lp.epsilon)

My only issue with this style is, if someone reorders this if-else block, then tests will fail. Capturing the block wiht if (_Greedy) catch makes it more explicit, and then implicit order above. 

Plus, that's consistent with how (_Linear) catches among different regressions.

Explicity is better than implicit. So I recommended keeping it as is